test/engine-luatest/pagination_test.lua

```diff
@@ -924,23 +923,3 @@ no_sup.after_each(function(cg)
         box.space.s:drop()
     end)
 end)
-
-no_sup.test_unsupported_pagination = function(cg)
-    cg.server:exec(function(index_type)
-        t.assert_error_msg_contains('does not support pagination',
-                box.space.s.index.sk.select, box.space.s.index.sk,
-                nil, {fullscan=true, fetch_pos=true})
-        local tuple = {0, 0}
-        if index_type == 'rtree' then
-            tuple = {0, {0, 0}}
-        end
```

Comment:
Why did you drop the test completely? Please, drop only these lines about `rtree`.